src/cli.zig

```diff
@@ -199,7 +226,19 @@ pub fn parse_args(allocator: std.mem.Allocator) !Command {
                         allocator,
                         addresses orelse fatal("required: --addresses", .{}),
                     ),
-                    .memory = if (memory) |m| parse_size(m) else config.memory_size_max_default,
+                    .cache_accounts = if (cache_accounts) |it|
+                        parse_size_to_count(tigerbeetle.Account, it)
+                    else
+                        config.cache_accounts_max,
+                    .cache_transfers = if (cache_transfers) |it|
+                        parse_size_to_count(tigerbeetle.Transfer, it)
+                    else
+                        config.cache_transfers_max,
+                    // TODO Is it `pending` or `posted`?
+                    .cache_transfers_posted = if (cache_transfers_posted) |it|
+                        parse_size_to_count(tigerbeetle.Transfer, it)
+                    else
+                        config.cache_transfers_pending_max,
```

Comment:
the layout here looks rather ugly, open to suggestions

Reply:
This is fixed in the latest version, but I have a Q about current code: here I use `|it|` as a name for "thing I don't care to name". This is idiomatic Kotlin. do we do this in tigerbeetle, or do we come up with reasonable names even for such through-away things?

Reply:
We sometimes use `it` for "iterator" (albeit inconsistently — often we spell out "iterator"), but not for general throwaway values. For optional values that are immediately used like in this case I would probably pick a short-but-full-word name, e.g. `size`.